packages/Php/tests/Rector/BinaryOp/IsCountableRector/IsCountableRectorTest.php

```diff
@@ -9,7 +9,11 @@ final class IsCountableRectorTest extends AbstractRectorTestCase
 {
     public function test(): void
     {
-        $this->doTestFiles([__DIR__ . '/Fixture/fixture.php.inc']);
+        if (PHP_VERSION_ID < 70300 && ! function_exists('is_countable')) {
```

Comment:
This is required cause some of Symfony dependencies force installation of polyfill on `dev` instalation of Rector.

Reply:
I see. The PHP version is taken from the config though, not real PHP version. So tests are consistent on various PHP versions.

See this: https://github.com/rectorphp/rector/pull/1674/files#diff-18a25bf46cd398e8a56a8ef9bf4f7172R26

You change PHP version to your desired one and this should be 2 tests then.

Reply:
If it's not possible, add a comment you wrote there, so we know "why" later